contributed_definitions/nyaml/NXmpes.yaml

```diff
@@ -135,107 +337,345 @@ NXmpes(NXobject):
         exists: optional
         doc: |
           Manipulator for positioning of the sample.
-        sample_temperature(NX_FLOAT):
+        temperature_sensor(NXsensor):
           exists: recommended
```

Comment:
maybe optional

Reply:
The idea was that sensors that are recommended in sample environments should _also_ be recommended in NXinstrument.

For example, since `NXsample/temperature(NXenvironment)` is recommended (and thus also `NXsample/temperature/temperature_sensor` _must_ be recommended because it stores the actual data), the `temperature_sensor` in `NXinstrument/NXmanipulator` should also be recommended. The actual data is then to be stored on `instrument/manipulator/temperature_sensor` and linked inside `sample/temperature/temperature_sensor`.

We can also change this such that , but to me the present design was most logical. Same is true for similar comments below

Reply:
Sure, it's a minor details, and I agree that a temperature measurement is good to have for an MPE experiment.